2023/10/28 11:35:58 failed to dial tcp connection, retrying
[2023-10-28T11:35:59.032Z] info  code-server 4.11.0 85e083580dec27ef19827ff42d3c9257d56ea7e3
[2023-10-28T11:35:59.034Z] info  Using user-data-dir ~/.local/share/code-server
[2023-10-28T11:35:59.047Z] info  Using config file ~/.config/code-server/config.yaml
[2023-10-28T11:35:59.047Z] info  HTTP server listening on http://127.0.0.1:8080/
[2023-10-28T11:35:59.047Z] info    - Authentication is disabled
[2023-10-28T11:35:59.047Z] info    - Not serving HTTPS
====================================>token: 69cc67ec-dc5f-43dc-8046-eac0a3e6b65f
[11:36:00] 




[11:36:00] Extension host agent started.
[11:36:02] [127.0.0.1][ec215990][ManagementConnection] New connection established.
[11:36:02] [127.0.0.1][e0d74df5][ExtensionHostConnection] New connection established.
[11:36:02] [127.0.0.1][e0d74df5][ExtensionHostConnection] <378> Launched Extension Host Process.
[11:36:06] Using the in-memory credential store as the operating system's credential store could not be accessed. Please see https://aka.ms/vscode-server-keyring on how to set this up. Details: libsecret-1.so.0: cannot open shared object file: No such file or directory
[11:36:15] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /opt/conda/lib/python3.8)
[2023-10-28T11:38:13.046Z] error Failed to get latest version [38;2;140;140;140m{"error":"connect ETIMEDOUT 20.205.243.168:443"}[0m
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x8986f1]

goroutine 271 [running]:
http_reverse_proxy/client.sendHttpRequestAndReplyHttpResponse(0x0?, 0x0?, 0xc0003a6040, 0xc0000ee060)
	/builds/hitminer/http_reverse_proxy/client/http_client.go:51 +0xb1
created by http_reverse_proxy/client.startHttpClient
	/builds/hitminer/http_reverse_proxy/client/http_client.go:34 +0x1ea
